lib/internal/bootstrap/node.js

```diff
@@ -156,13 +156,31 @@ const rawMethods = internalBinding('process_methods');
   process._getActiveHandles = rawMethods._getActiveHandles;
 
   process.getActiveResourcesInfo = function() {
+    let timeoutCount = 0;
+    const lists = ObjectValues(internalTimers.timerListMap);
```

Comment:
Do you care about perf here? If so, it should be faster to just do `for..in` rather than `ObjectValues` followed by a for loop.